docker/my-dojo/overrides/bitcoind.install.yaml

```diff
@@ -24,6 +24,7 @@ services:
     volumes:
       - data-bitcoind:/home/bitcoin/.bitcoin
       - data-tor:/var/lib/tor
+      - ${BITCOIND_BLOCKS_DIR:-/dev/null}:/home/bitcoin/blocks
```

Comment:
Wouldn't this send all blocks to `/dev/null` if `BITCOIND_BLOCKS_DIR` is not specified? And also try to read from `/dev/null`?

Reply:
But the check in `restart.sh` is `if [ "$BITCOIND_BLOCKS_DIR" != "" ]`. So that should probably be `if [ "$BITCOIND_BLOCKS_DIR" != "/dev/null" ]` as you had it previously?

Reply:
The default is an empty string now inside docker-bitcoind.conf.tpl but I can change it back to /dev/null if you find it better

Reply:
No, this is actually good. I will merge it and test it. Thank you